### DISCUSSION TITLE
Can I use tfc.entropy_models to estimate the bits for fixed model?

### DISCUSSION THREAD

**Discussion by gitlabspy** [tensorflow/compression#150] [Q&A] [answered]
Hello everyone!
Can I use tfc.entropy_models to estimate the bits/bpp of fixed model's output, like JPEG encoder's output or some other pretrained image compression encoder's output?

**Comment by jonaballe** [comment, marked as answer]
I'm not sure what you are asking. If you already have an encoder, why not just encode the image and measure the number of bits of the encoded image?

What you could do is train a prior distribution on an existing latent representation (for example the DCT coefficients in JPEG) and then use the TFC entropy model class to estimate the bits/bpp after quantization. But that would ignore/replace the entropy coding scheme that is defined by JPEG.
